cmd/lncli/commands.go

```diff
@@ -2387,12 +2387,26 @@ func updateChannelPolicy(ctx *cli.Context) error {
 		return errors.New("inbound_fee_rate_ppm out of range")
 	}
 
+	// Inbound fees are optional. However, if an update is required,
+	// both the base fee and the fee rate must be provided.
+	var inboundFee *lnrpc.InboundFee
+	switch {
```

Comment:
I'd remove the switch case and do the following instead, easier to understand,
```go
	if ctx.IsSet("inbound_base_fee_msat") !=
		ctx.IsSet("inbound_fee_rate_ppm") {

		return errors.New("both parameters must be provided: " +
			"inbound_base_fee_msat and inbound_fee_rate_ppm")
	}

	inboundFee := &lnrpc.InboundFee{
		BaseFeeMsat: int32(inboundBaseFeeMsat),
		FeeRatePpm:  int32(inboundFeeRatePpm),
	}
```

Reply:
Thank you very much for your comments. I'll take a detailed look tomorrow. 
At this point I don't quite understand what you mean. We need to catch the case where both lncli are not set and then pass nil to the rpc. Or am I missing something?

Reply:
nvm, it seems this PR requires the user to set both `inbound_fee_rate_ppm` and `inbound_base_fee_msat` at the same time, but I think we want to allow updating a single field, so either `inbound_base_fee_msat` or  `inbound_fee_rate_ppm`?